redis/server/clean.sls

```diff
@@ -0,0 +1,44 @@
+
+{%- from "redis/map.jinja" import redis_settings with context %}
+{%- set cfg_name          = redis_settings.cfg_name -%}
+{%- set install_from      = redis_settings.install_from -%}
+{%- set svc_name          = redis_settings.svc_name -%}
+{%- set port              = redis_settings.port -%}
+
+redis-server-service-clean:
+  service.dead:
+    {%- if install_from == 'source' %}
+    - name: {{ svc_name }}_{{ port }}
+    {%- else %}
+    - name: {{ svc_name }}
+    {%- endif %}
+    - enable: false
+    - require_in:
+      - user: redis-server-clean
+      - file: redis-server-clean
+
+redis-server-clean:
+    {%- if install_from == 'source' %}
+        {%- set user = redis_settings.user -%}
+        {%- set group = redis_settings.group -%}
```

Comment:
Is there a reason the indentation is does not start at column 0?

Reply:
It's my generational style (what I always do) when mixing A&B language I want to visually or stylistically differentiate, a very ingrained pattern I lean towards.